### DISCUSSION TITLE
Github Education pack not active

### DISCUSSION THREAD

**Discussion by ee24-s031** [community/community#135034] [GitHub Education] [answered]
### Hi everyone,

I got Verified on Aug 3, 2024 But The education pack is not active

**Comment by UjjwalSaini07** [comment]
Its not a problem if u receive the mail that ur application is accepted so u got Student developer pack in which u enjoy pro account services.
Sometimes it take long time due to server glitch or workload.
i also get developer pack on my 6th day so dont worry and wait u definately got that one.

if u suppose this comment helps u well so mindly do **_correct answer_** to this comment this really helpful for mine and anyone else who has problem related to this, gets well
Thanx!!!

**Reply by ee24-s031** [reply]
I don't have access to any pro features it seems, I tried using co-pilot but it says I have to subscribe for it.
If I am not wrong education pack comes with co-pilot.

**Reply by UjjwalSaini07** [reply]
Yes When u get proper Student Developer pack in your account so u are able to use all pro features

**Reply by UjjwalSaini07** [reply]
When in ur GitHub Profile 
![image](https://github.com/user-attachments/assets/0d31afcd-cd0b-4dec-88db-d1dc7c157e04)
this pro is visible so it means that u are now fully eligible for using all services.

So Wait is the only Solution for that.

**Reply by ee24-s031** [reply]
@UjjwalSaini07 
I can find this in my application
![Screenshot from 2024-08-08 12-00-21](https://github.com/user-attachments/assets/ac25956e-6475-4130-8a10-bd8af396c5ea)

**Reply by UjjwalSaini07** [reply]
yes Great means u fulfil all requirements for getting student developer pack.
**_U Definitely get one, today itself or up to Saturday._**

If anyone fail in application so it hard for getting benefits but u fulfil all requirement so u get that. So dont worry 👍

**Reply by ee24-s031** [reply]
I agree with your statement but I got this approved status on Aug 3, 2024 and still didn't get access to any service. By any chance do you know  how long it usually takes

**Reply by UjjwalSaini07** [reply, marked as answer]
max to max 2 weeks
I said previously that approved is only for confirmation when u got mail as pro user so u able to use all services

**Reply by UjjwalSaini07** [reply]
Thanx a lot for giving marked as answer
if more doubts so feel free to ask

**Reply by rohanvkumarv** [reply]
i am facing same issue mine was like approved on 31st july still no access of perks till now

**Reply by UjjwalSaini07** [reply]
If u got approved msg so wait is only solution u definitely got the perks and benefits of pro GitHub user
and mail like this: 
![image](https://github.com/user-attachments/assets/087649a5-fb4a-422a-a4f3-a2fcdc664483)

**Reply by ee24-s031** [reply]
> i am facing same issue mine was like approved on 31st july still no access of perks till now

Hey I got My Mail Just now, Maybe you have to re-apply or contact GitHub support

**Reply by rohanvkumarv** [reply]
what you did that you got the mail ? did u re apply ? or contacted github support? if u contacted github support how do i do that  ?

**Reply by ee24-s031** [reply]
> what you did that you got the mail ? did u reply ? or contacted github support? if u contacted github support how do i do that ?

I didn't contact support, I did nothing to get it. 
I did contact support for different issue through this mail-id "support@githubsupport.com"

**Reply by rohanvkumarv** [reply]
oh okay okay

**Reply by rafiqcoder01** [reply]
I am facing same issue

**Reply by rafiqcoder01** [reply]
![image](https://github.com/user-attachments/assets/bfc0e025-8630-4b6a-aa24-b45ce9c0e234)

**Reply by UjjwalSaini07** [reply]
please dont give request again and again might be blocked u to making spam do wait please
if take a case that u gave 1st request on 2aug and the process complete on 10th but u re-request on 8th so it overwrite and take again 7 to 8 days so this is a rubbish method if u do all process correctly so wait is the only option.
Please not do that thing which u do above it treated as spam.

**Comment by Shubham-Padhi** [comment]
Response #1
